app.js

```diff
@@ -1,8 +1,80 @@
-// import functions and grab DOM elements
+const ball1 = document.getElementById('ball-1');
+const ball2 = document.getElementById('ball-2');
+const ball3 = document.getElementById('ball-3');
 
-// let state
+const button1 = document.getElementById('shell-1-button');
+const button2 = document.getElementById('shell-2-button');
+const button3 = document.getElementById('shell-3-button');
 
-// set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+const winSpan = document.getElementById('wins');
+const lossSpan = document.getElementById('losses');
+const totalSpan = document.getElementById('total');
+
+let wins = 0;
+let total = 0;
+
+function removeBall(){
+    ball1.classList.remove('reveal');
+    ball2.classList.remove('reveal');
+    ball3.classList.remove('reveal');
+}
+
+button1.addEventListener('click', () => {
+    removeBall();
+    total++;
```

Comment:
In order to avoid repeating yourself in code, you'd want the logic for incrementing your player rounds to happen in the separate function that you use to check if the guess was correct.